nsupdate.sh

```diff
@@ -38,7 +38,7 @@ if ls $(dirname $0)/nsupdate.d/*.config &> /dev/null; then
    for f in $(dirname $0)/nsupdate.d/*.config
    do
       if [[ "$SILENT" == "NO" ]]; then
-         echo "Starting nameserver update with config file $f"
+         echo "Starting nameserver update with config file $f ($LOG)"
```

Comment:
When you override `LOG` in the config file this will log the wrong file (e.g. the default log file for the first config).

Reply:
Fixed in new commit.